rules/windows/process_creation/proc_creation_win_attrib_hiding_files.yml

```diff
@@ -4,24 +4,26 @@ status: test
 description: Detects usage of attrib.exe to hide files from users.
 author: Sami Ruohonen
 date: 2019/01/16
-modified: 2021/11/27
+modified: 2022/11/11
 tags:
     - attack.defense_evasion
     - attack.t1564.001
 logsource:
     category: process_creation
     product: windows
 detection:
-    selection:
-        Image|endswith: '\attrib.exe'
+    selection_img:
+        - Image|endswith: '\attrib.exe'
+        - OriginalFileName: 'ATTRIB.EXE'
+    selection_cli:
         CommandLine|contains: ' +h '
-    ini:
+    filter_ini:
         CommandLine|contains: '\desktop.ini '
-    intel:
+    filter_intel:
         ParentImage|endswith: '\cmd.exe'
         CommandLine: +R +H +S +A \\*.cui
         ParentCommandLine: C:\WINDOWS\system32\\*.bat
```

Comment:
`\\*` not sure of the escape result.
we are looking for `\*.bat` or `\*.cui`, for me it is more
```yaml
        CommandLine: +R +H +S +A \\\*.cui
        ParentCommandLine: C:\\WINDOWS\\system32\\\*.bat
```

Reply:
You could write it like that for clarity. But it's correct as it is I think. Because the `\` doesn't need to be escaped, you only need to escape the `*`

I will change it for consistency

Reply:
Yes , but can be ticky depending on the converter.
Better keep the consistency when escape